src/setData/yearOne.js

```diff
@@ -536,13 +513,7 @@ export default ([
             name: 'Emblems',
```

Comment:
Could you change this to "Extras" pelase?

Reply:
Can't believe I missed one out of them all haha